source/braille.py

```diff
@@ -1211,6 +1213,20 @@ def _set_windowEndPos(self, endPos):
 		if region.focusToHardLeft:
 			# Only scroll to the start of this region.
 			restrictPos = endPos - regionPos - 1
+		elif config.conf["braille"]["focusContextPresentation"]==2:

```

Comment:
If we'd go for constants, I'd say we should go for strings, or for 
option to be super modern. Items like tethering in the config spec 
should be option type as well, actually. Unless you can't set a default 
for those.